<?xml version="1.0"?>
<launch>
  <arg name="object_name" default="HV8" />
  <arg name="icpRefine" default="False" />
  <arg name="arch" default="PointNet_Pose" />
  <arg name="name" default="PointNet" />
  <arg name="resolution" default="50" />
  <arg name="load_path" default="/home/tsuchidashinya/MEGAsync/X10/day_1/PointNet/dataset_20000.hdf5/70_net.pth"/>
  <node name="random_move" pkg="gen_dataset" type="random_move_euler.py"/>
  <node name="estimator_client" pkg="estimator" type="estimator_client.py" output="screen" >
    <param name="object_name" value="$(arg object_name)" />
    <param name="icpRefine" value="$(arg icpRefine)" />
    <param name="arch" value="$(arg arch)" />
    <param name="resolution" value="$(arg resolution)" />
  </node>

  <node name="estimator_server" pkg="estimator" type="estimator_server.py" output="screen" >
    <param name="object_name" value="$(arg object_name)" />
    <param name="name" value="$(arg name)" />
    <param name="arch" value="$(arg arch)" />
    <param name="resolution" value="$(arg resolution)" />
    <param name="load_path" value="$(arg load_path)"/>
  </node>

  
</launch>
